### PR TITLE
Move the element type check out of the ready check

### PR DESCRIPTION
**Pull request by foolip**
There is no need to check this again in the animation frame task.

### CODE REVIEW

File: fullscreen.bs [hunk 1]
```diff
@@ -208,14 +202,20 @@ these steps:
  <li><p>Let <var>promise</var> be a new promise.
 
  <li>
-  <p>If any of the following conditions are true, set <var>error</var> to true:
+  <p>If any of the following conditions are false, set <var>error</var> to true:
```
Comment (annevk) [inline fullscreen.bs]: then set*

**Comment by annevk** [comment]
LGTM with that.

**Comment by foolip** [comment]
I also fixed two other instances, can you confirm that is what you had in mind?

**Comment by foolip** [comment]
Test: https://github.com/w3c/web-platform-tests/pull/3830

**Pull request merged**